lib/factor/factorQuadratic.js

```diff
@@ -192,22 +197,62 @@ function factorPerfectSquare(node, symbol, aValue, bValue, cValue, negate) {
   return Node.Status.noChange(node);
 }
 
-// Will factor the node if it's in the form of x^2 + bx + c (i.e. a is 1), by
-// applying the sum product rule: finding factors of c that add up to b.
-// e.g. x^2 + 3x + 2 -> (x + 1)(x + 2)
+// Will factor the node if it's in the form of x^2 + bx + c, by
+// applying the sum product rule: finding factors of a*c that add up to b.
+// e.g. x^2 + 3x + 2 -> (x + 1)(x + 2) or
+// or   2x^2 + 5x + 3 -> (2x - 1)(x + 3)
 function factorSumProductRule(node, symbol, aValue, bValue, cValue, negate) {
```

Comment:
I agree, but will leave a comment for this to be added at a later time (with an appropriate issue). I looked into doing this initially and it wasn't simple. Similarly above, I think that factoring out the gcd should be a substep for the other functions.